src/helpers/index.js

```diff
@@ -0,0 +1,711 @@
+/**
+ * @module helpers
+ */
+
+/**
+ * Earth Radius used with the Harvesine formula and approximates using a spherical (non-ellipsoid) Earth.
+ *
+ * @memberof helpers
+ * @type {number}
+ */
+export let earthRadius = 6371008.8;
+
+/**
+ * Unit of measurement factors using a spherical (non-ellipsoid) earth radius.
+ *
+ * @memberof helpers
+ * @type {Object}
+ */
+export let factors = {
+    centimeters: earthRadius * 100,
+    centimetres: earthRadius * 100,
+    degrees: 180 / Math.PI, // See https://github.com/Turfjs/turf/issues/1406
+    feet: earthRadius * 3.28084,
+    inches: earthRadius * 39.370,
+    kilometers: earthRadius / 1000,
+    kilometres: earthRadius / 1000,
+    meters: earthRadius,
+    metres: earthRadius,
+    miles: earthRadius / 1609.344,
+    millimeters: earthRadius * 1000,
+    millimetres: earthRadius * 1000,
+    nauticalmiles: earthRadius / 1852,
+    radians: 1,
+    yards: earthRadius / 1.0936,
+};
+
+/**
+ * Units of measurement factors based on 1 meter.
+ *
+ * @memberof helpers
+ * @type {Object}
+ */
+// export let unitsFactors = {
```

Comment:
loose end here: unitsFactors is specified in the typescript definition, but it's now commented-out.

Reply:
I can't actually work out where this gets used... from what I can see it's a relatively new addition. `factors` above it and `areaFactors` below it both get used. So I'm going to delete it